lib/lotus/utils/kernel.rb

```diff
@@ -1021,13 +1021,40 @@ def self.Pathname(arg)
       #   Lotus::Utils::Kernel.Symbol(input) # => TypeError
       def self.Symbol(arg)
         case arg
-        when '' then raise TypeError.new "can't convert into Symbol"
+        when '' then raise TypeError.new "can't convert #{inspect_type_error(arg)}into Symbol"
         when ->(a) { a.respond_to?(:to_sym) } then arg.to_sym
         else
-          raise TypeError.new "can't convert into Symbol"
+          raise TypeError.new "can't convert #{inspect_type_error(arg)}into Symbol"
         end
       rescue NoMethodError
-        raise TypeError.new "can't convert into Symbol"
+        raise TypeError.new "can't convert #{inspect_type_error(arg)}into Symbol"
+      end
+
+      class << self
+        # Returns the most useful type error possible
+        #
+        # If the object does not respond_to?(:inspect), we return the class, else we
+        # return nil. In all cases, this method is tightly bound to callers, as this
+        # method appends the required space to make the error message look good.
+        #
+        # Used internally by Lotus::Utils::Kernel. This method is not part of the public
+        # API.
+        #
+        # @since 0.4.2
```

Comment:
please, let @jodosha put 0.4.2 better for now x.x.x, and if you put a space between `@since` and `@private` I'll thank you